spec/models/manageiq/providers/automation_manager_spec.rb

```diff
@@ -0,0 +1,4 @@
+describe ManageIQ::Providers::AutomationManager do
+  let(:provider)           { FactoryGirl.build(:provider) }
+  let(:automation_manager) { FactoryGirl.build(:automation_manager, :provider => provider) }
+end
```

Comment:
👍 

🙏 Please squash your commits when it's ready, this is *quite* a many rough commits to follow in history later on. 

![](http://screenshots.chrisarcand.com/permedcfg.jpg)